platform/logging/LoggingConfig.ts

```diff
@@ -0,0 +1,166 @@
+import { Schema, typeOfSchema } from '../types';
+import { LogLevelId } from './LogLevel';
+import { Appenders, AppenderConfigType } from './appenders/Appenders';
+
+/**
+ * Separator string that used within nested context name (eg. plugins::pid).
+ */
+const CONTEXT_SEPARATOR = '::';
+
+/**
+ * Name of the `root` context that should always be defined.
+ */
+const ROOT_CONTEXT_NAME = 'root';
+
+const createLoggerSchema = (schema: Schema) => {
+  const { literal, object, string, arrayOf, oneOf } = schema;
+
+  return object({
+    level: oneOf(
+      [
+        literal('all'),
+        literal('fatal'),
+        literal('error'),
+        literal('warn'),
+        literal('info'),
+        literal('debug'),
+        literal('trace'),
+        literal('off')
+      ],
+      {
+        defaultValue: 'info'
+      }
+    ),
+    appenders: arrayOf(string(), { defaultValue: [] })
+  });
+};
+
+const createLoggingSchema = (schema: Schema) => {
+  const { object, string, mapOf } = schema;
+
+  return object({
+    appenders: mapOf(string(), Appenders.createConfigSchema(schema), {
+      defaultValue: new Map([
```

Comment:
These defaults are just "something" to have empty config and working logging at the same time, we should probably discuss (here or in the follow-up) what defaults do we really want.